test/arranger-conduit/ViewFunctions.t.sol

```diff
@@ -1,23 +1,125 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity ^0.8.13;
 
-import { UpgradeableProxy } from "../../lib/upgradeable-proxy/src/UpgradeableProxy.sol";
-
 import { console2 as console } from "../../lib/forge-std/src/console2.sol";
```

Comment:
Unused. Btw I notice you forgetting to remove this a lot. Usually what I do is just do a full import of the testing infrastructure as it brings a lot of the tools needed in-scope. So would be just `import "forge-std/Test.sol"` instead of just a few contracts.